### ISSUE TITLE
Bug with /sign ui with Paper 1.17

### ISSUE THREAD

**Issue by aurel85** [Deltik/SignEdit#21]
Hi,

With Paper 1.17 the command /sign ui doesn't work anymore.

Error in the console:
```
21.06 11:26:14 [Server] INFO aurel85 issued server command: /sign ui
21.06 11:26:14 [Server] ERROR java.lang.IllegalAccessException: Can not set static final boolean field net.minecraft.world.level.block.entity.TileEntitySign.CONVERT_LEGACY_SIGNS to java.lang.Boolean
21.06 11:26:14 [Server] INFO at java.base/jdk.internal.reflect.UnsafeFieldAccessorImpl.throwFinalFieldIllegalAccessException(UnsafeFieldAccessorImpl.java:76)
21.06 11:26:14 [Server] INFO at java.base/jdk.internal.reflect.UnsafeFieldAccessorImpl.throwFinalFieldIllegalAccessException(UnsafeFieldAccessorImpl.java:80)
21.06 11:26:14 [Server] INFO at java.base/jdk.internal.reflect.UnsafeQualifiedStaticBooleanFieldAccessorImpl.set(UnsafeQualifiedStaticBooleanFieldAccessorImpl.java:77)
21.06 11:26:14 [Server] INFO at java.base/java.lang.reflect.Field.set(Field.java:793)
21.06 11:26:14 [Server] INFO at org.deltik.mc.signedit.interactions.UiSignEditInteraction.interact(UiSignEditInteraction.java:3152)
21.06 11:26:14 [Server] INFO at org.deltik.mc.signedit.commands.SignCommand.onCommand(SignCommand.java:1133)
21.06 11:26:14 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
21.06 11:26:14 [Server] INFO at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
21.06 11:26:14 [Server] INFO at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:821)
21.06 11:26:14 [Server] INFO at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:2185)
21.06 11:26:14 [Server] INFO at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1996)
21.06 11:26:14 [Server] INFO at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1977)
21.06 11:26:14 [Server] INFO at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46)
21.06 11:26:14 [Server] INFO at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:6)
21.06 11:26:14 [Server] INFO at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$1(PlayerConnectionUtils.java:36)
21.06 11:26:14 [Server] INFO at net.minecraft.server.TickTask.run(TickTask.java:18)
21.06 11:26:14 [Server] INFO at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:149)
21.06 11:26:14 [Server] INFO at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(IAsyncTaskHandlerReentrant.java:23)
21.06 11:26:14 [Server] INFO at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:1340)
21.06 11:26:14 [Server] INFO at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:193)
21.06 11:26:14 [Server] INFO at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:122)
21.06 11:26:14 [Server] INFO at net.minecraft.server.MinecraftServer.bg(MinecraftServer.java:1319)
21.06 11:26:14 [Server] INFO at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1312)
21.06 11:26:14 [Server] INFO at net.minecraft.util.thread.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:132)
21.06 11:26:14 [Server] INFO at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1273)
21.06 11:26:14 [Server] INFO at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1184)
21.06 11:26:14 [Server] INFO at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320)
21.06 11:26:14 [Server] INFO at java.base/java.lang.Thread.run(Thread.java:831)
21.06 11:26:26 [Multicraft] Skipped 1 lines due to rate limit (30/s)
```

The other things I tested are ok.

I use the last version of this plugin (1.12.7).

Thank you in advance.

**Comment by Deltik** [comment]
@aurel85: Thanks for reporting this bug!

It has now been fixed in [SignEdit for Bukkit v1.12.8](https://github.com/Deltik/SignEdit/releases/tag/v1.12.8), which you can directly download [here](https://github.com/Deltik/SignEdit/releases/download/v1.12.8/SignEdit-1.12.8.jar).

**Comment by aurel85** [comment]
I tested and everything is ok, thank you !
